packages/@vuepress/theme-default/src/client/components/ToggleDarkButton.vue

```diff
@@ -44,17 +44,39 @@ const toggleDark = (): void => {
   isDark.value = !isDark.value
 }
 
-const setDarkClass = (): void => {
+const updateDarkClass = (): void => {
   const htmlEl = window?.document.querySelector('html')
   htmlEl?.classList.toggle('dark', isDark.value)
 }
 
+const updateThemeColor = (): void => {
+  const themeColorEl = window?.document.querySelectorAll(
+    'meta[name="theme-color"]'
+  )
+  if (themeColorEl) {
+    const newThemeColor = window
+      .getComputedStyle(window.document.documentElement)
+      ?.getPropertyValue(isDark.value ? '--c-bg' : '--c-brand')
+      .trim()
+    if (newThemeColor) {
+      themeColorEl.forEach((mata) => {
+        mata.setAttribute('content', newThemeColor)
+      })
+    }
+  }
+}
+
+const setDarkMode = (): void => {
```

Comment:
`setAppearance` or `setThemeMode` could be better

Reply:
Done in https://github.com/vuepress/vuepress-next/pull/218/commits/c964abcf35922bbee773777fef203c90e965d705